git-ls-files: add page (#4753)
tldr-pages/tldr: pages/common/git-ls-files.md

# git ls-files

> > Git command for showing information about files in the index and the working tree.

Reviewer: Perhaps this will read better?
```suggestion
> Show information about files in the index and the working tree.
```

Author: @sbrl I do agree that this is better, however notice that some git commands have descriptions like this, and this is where i get this template from :)

# git ls-files

> Git command for showing information about files in the index and the working tree.
> More information: <https://git-scm.com/docs/git-ls-files>.

> - Show deleted files in the output:

Reviewer 2: ```suggestion
- Show deleted files:
```

`git ls-files --deleted`

> - Show modified files in the output - deleted files are also part of the modified files:

Reviewer 2: Maybe this is better:
```suggestion
- Show modified and deleted files:
```

`git ls-files --modified`

- Show ignored and untracked files:

`git ls-files --others`

> 
> 
> 

Reviewer 2: You should delete this empty lines to pass the linter check:
```suggestion
```
